samples/hello_world/src/main.c

```diff
@@ -3,11 +3,147 @@
  *
  * SPDX-License-Identifier: Apache-2.0
  */
+// Reading ESP32 CPU Clock freq
```

Comment:
Obviously this can't go into hello_world.  Consider making it a separate sample, or better yet a test of the clock_control driver that can be implemented on other platforms.

Reply:
I agree, This is committed by mistake.
The whole sample here is about reading clock registers and print current clock configuration.

What do you think if I add it to `HWInfo` module for esp32 ?
The Idea for test is great, but I think it will take me a lot of time to figure out a design which is SoC independent.